src/main/java/core/basesyntax/service/impl/CsvFileWriterService.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvFileWriterService implements FileWriterService {
+    private static final String DEFAULT_NAME = "report";
```

Comment:
You use this 2 constant only in combination, maybe it makes sence to make it 1 constant